### PR TITLE
Add DYS F4 4530D unified target. http://www.dys.hk/product/648.html#

### PR DESCRIPTION
**Pull request by apeelme**

### CODE REVIEW

File: configs/default/DYST-DYSF44530D.config [hunk 1]
```diff
@@ -0,0 +1,133 @@
+board_name DYSF44530D
```
Comment (mikeller) [inline configs/default/DYST-DYSF44530D.config:1]: Please read and follow the instructions on https://github.com/betaflight/unified-targets (**including testing**) - this was clearly not done by following the instructions, as it is missing the first line.

File: configs/default/DYST-DYSF44530D.config [hunk 2]
```diff
@@ -0,0 +1,133 @@
...
+set beeper_inversion = ON
+set beeper_od = OFF
+set system_hse_mhz = 8
+set max7456_spi_bus = 2
+set gyro_1_bustype = SPI
+set gyro_1_spibus = 1
+set gyro_1_sensor_align = CW270
+set blackbox_device = SPIFLASH
+set flash_spi_bus = 3
+set mag_hardware = NONE
```
Comment (mikeller) [inline configs/default/DYST-DYSF44530D.config:132]: There is no reason to set this - leaving the default will work just fine with no magnetometer / barometer connected. If you set this, all users wanting to use a magnetometer / barometer will be advised to buy a new flight controller from a different manufacturer in order to get magnetometer / barometer support, as this is not supported by this hardware.

File: configs/default/DYST-DYSF44530D.config [hunk 3]
```diff
@@ -0,0 +1,133 @@
...
+set baro_hardware = NONE
```
Comment (mikeller) [inline configs/default/DYST-DYSF44530D.config:133]: There is no reason to set this - leaving the default will work just fine with no magnetometer / barometer connected. If you set this, all users wanting to use a magnetometer / barometer will be advised to buy a new flight controller from a different manufacturer in order to get magnetometer / barometer support, as this is not supported by this hardware.

File: configs/default/DYST-DYSF44530D.config [hunk 4]
```diff
@@ -0,0 +1,133 @@
...
+dma pin B06 0
+# pin B06: DMA1 Stream 0 Channel 2
+dma pin A00 0
+# pin A00: DMA1 Stream 2 Channel 6
+dma pin A01 0
+# pin A01: DMA1 Stream 4 Channel 6
+
+# feature
+feature -RX_PARALLEL_PWM
+feature -SOFTSERIAL
```
Comment (mikeller) [inline configs/default/DYST-DYSF44530D.config:107]: Please follow the instructions - default settings are not permissible in Unified Target configurations.

**Review by mikeller — CHANGES_REQUESTED** [review]
@apeelme: Are you the official target maintainer for the DYS manufacturer?

**Comment by apeelme** [comment]
> @apeelme: Are you the official target maintainer for the DYS manufacturer?

No.

**Review by mikeller — APPROVED** [review]

**Pull request merged**